<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSetDialog</class>
 <widget class="QDialog" name="ColorSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>最冷色</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>最暖色</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cool">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>#ff00ff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_warm">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>#ff0000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="XGradientSlider" name="Slider_cool">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="XGradientSlider" name="Slider_warm">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="value">
    <number>99</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XGradientSlider</class>
   <extends>QSlider</extends>
   <header location="global">xgradientslider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
